Features/Stories/FieldTypes/TextlineFieldType.feature

```diff
@@ -0,0 +1,246 @@
+Feature: Test the validations done on fields from PlatformUI - text line fieldtype
+    In order to validate the text line fieldtype
+    As an Editor user
+    I need to be able to create and update content with text line fieldtypes
+
+    Background:
+       Given I am logged in as an 'Administrator' in PlatformUI
+
+    ##
+    # Validate the existence of expected fields from a field type when creating a content
+    ##
+    @javascript @common
+    Scenario: A Content of an Content Type that has a text line fieldtype must have a text field
```

Comment:
`Scenario: A Content of an Content Type having a Text Line Field Definition must have a Text Line Field`